spec/controllers/games_controller_spec.rb

```diff
@@ -21,9 +21,25 @@
       expect(response).to redirect_to new_user_session_path
     end
   end
+
   describe 'games#create' do
     it 'should create a new game in the database' do
       process :create, method: :post, game: { turn: nil }
     end
   end
+
+  describe 'games#update' do
+    it 'should update the coords of the white pawn in the database' do
+      game = FactoryGirl.create(:game)
+      # test not working currently, something is wrong with its params. To be fixed.
+      patch :update, id: game.id, game: { piece_id: 24, x_position: 7, y_position: 5 }
```

Comment:
@lasley this is the line that we have the problem with. And this is the error that we get:

```Ruby
GamesController games#update should update the coords of the white pawn in the database
     Failure/Error: @piece = @game.pieces.find(params[:piece_id])

     ActiveRecord::RecordNotFound:
       Couldn't find Piece without an ID
```